Funky-Chickens/workFORCEmanagement
# Departments jmr

shows all departments

shows department details
 -shows all employees
 -shows supervisor 

reformatted lists so they are uniform and look cleaner

to test: npm start

click on the departments tab in the navbar

click on every department and it should show you associated supervisor name and employee name.
